plugin-server/src/utils/db/db.ts

```diff
@@ -840,19 +840,24 @@ export class DB {
 
         const values = [...updateValues, person.id]
 
+        // Potentially overriding values badly if there was an update to the person after computing updateValues above
```

Comment:
This comment does not make sense.

Reply:
what's not clear about it, I'm referring to the not solved issue (1) in the description